src/UnityExtension/Assets/Editor/GitHub.Unity/UI/GitPathView.cs

```diff
@@ -0,0 +1,286 @@
+                            gitExecParent,
+                            gitExecExtension);
+
+                        if (!string.IsNullOrEmpty(newValue))
+                        {
+                            newGitExec = newValue;
+
+                            if (Environment.IsWindows)
+                            {
+                                //Normalizing the path separator in windows
+                                newGitExec = newGitExec.ToNPath().ToString();
+                            }
+
+                            CheckEnteredGitPath();
+                        }
+                    }
+                }
+                GUILayout.EndHorizontal();
+
+                GUILayout.Space(EditorGUIUtility.standardVerticalSpacing);
+
+                GUILayout.BeginHorizontal();
+                {
+                    EditorGUI.BeginDisabledGroup(!isValueChangedAndFileExists);
+                    {
+                        if (GUILayout.Button(GitPathSaveButton, GUILayout.ExpandWidth(false)))
+                        {
+                            GUI.FocusControl(null);
+                            isBusy = true;
+
+                            ValidateAndSetGitInstallPath(newGitExec);
+                        }
+                    }
+                    EditorGUI.EndDisabledGroup();
+
+                    //Find button - for attempting to locate a new install
+                    if (GUILayout.Button(GitInstallFindButton, GUILayout.ExpandWidth(false)))
+                    {
+                        GUI.FocusControl(null);
+                        isBusy = true;
+
+                        newGitExec = gitExec;
+                        CheckEnteredGitPath();
+
+                        new ProcessTask<NPath>(Manager.CancellationToken, new FirstLineIsPathOutputProcessor())
+                            .Configure(Manager.ProcessManager, Environment.IsWindows ? "where" : "which", "git")
+                            .FinallyInUI((success, ex, path) =>
+                            {
+                                if (success)
+                                {
+                                    Logger.Trace("FindGit Path:{0}", path);
+                                }
+                                else
+                                {
+                                    if (ex != null)
+                                    {
+                                        Logger.Error(ex, "FindGit Error Path:{0}", path);
+                                    }
+                                    else
+                                    {
+                                        Logger.Error("FindGit Failed Path:{0}", path);
+                                    }
+                                }
+
+                                if (success)
+                                {
+                                    newGitExec = path;
+                                    CheckEnteredGitPath();
```

Comment:
This block of code should probably be merged into the `if (success)` above?

Reply:
👍